guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -56,7 +83,7 @@ For more information on performing an incremental Library export, see xref:Using
 . Import content to an organization using the procedure outlined in xref:Using_ISS-Importing-Library[].
 You can then manage content using Content Views or Lifecycle Environments as you require on the disconnected {ProjectServer}.
 
-==== Disconnected
+==== Managed Content
```

Comment:
```suggestion
=== Importing Managed Content from a Connected to a Disconnected {ProjectServer}
```

Reply:
How about
```suggestion
==== CDN connected {ProjectServer} for Content Curation
```

or 

```suggestion
==== {ProjectServer} connected to the CDN as Content Curator
```

Reply:
This doesn't work as well. 
The problem is that "Content Curator" doesn't mean anything to anyone immediately?. 
What do you mean by content curator?
Can we be more explicit? 
The title needs to show the action that we talk about in this section. 
What is wrong with describing the Foreman server as connected? 
We are essentially setting up a disconnected Foreman to receive updates from a connected Foreman. On this connected Foreman, all the content management is done. 

```suggestion
==== Importing Curated Content from a Connected {Project}
```

How about that?

Reply:
This section is more about how you want your Export/Import setup to be rather than just say 'Importing' something vs 'Exporting' something.  I have updated slightly but open for suggestions :).